packages/core/src/components/select/select.tsx

```diff
@@ -960,6 +976,8 @@ export class Select implements IxInputFieldComponent<string | string[]> {
 
             return {
               minWidth: `${minWidth}px`,
+              width: `${this.dropdownWidth}rem`,
+              maxWidth: `${this.dropdownMaxWidth}rem`,
```

Comment:
If you allow here only number values and add a suffix `rem` to it prevents user from building response bahavior e.g 50vw or 50%